views/Query/Collapsed.jsx

```diff
@@ -55,8 +55,7 @@ export default React.createClass({
     this.setState({ focusedInput });
   },
 
-  onDatesChange({ startDate, endDate }) {
-    this.setState({ startDate, endDate });
+  onDatesChange() {
```

Comment:
onDatesChange has a signature change (removing the first arg) but is called with an argument on line 126.